src/graphql/queries/events.ts

```diff
@@ -34,11 +42,8 @@ export const GET_ALL_EVENTS = gql`
       id
       name
       eventDate
-      ambulances {
-        id
-        vehicleNumber
-      }
-      hospitals {
```

Comment:
i actually meant these ones should be added back too

Reply:
this PR is being kind of weird, I see those fields still exist locally. I'll push again

Reply:
here's the file: https://github.com/uwblueprint/paramedics-react/blob/bb54f43240da91fb8820d2fdeabf65103306780f/src/graphql/queries/events.ts